File: unlock.js
for (var j = 0; j < eth.accounts.length; j++) {
	console.log(web3.fromWei(eth.getBalance(eth.accounts[j]), "ether"));
}

for (var j = 0; j < eth.accounts.length; j++) {
	personal.unlockAccount(eth.accounts[j], "cuentahoracio0", 22000);
}

File: deployment/unlock.py
from web3_interface import Web3Interface

class Unlock:
  
  web3 = None
  
  def __init__(self):
    self.web3 = Web3Interface("middleware").w3
  
  def unlock(self):
    for x in self.web3.eth.accounts:
      self.web3.personal.unlockAccount(x, 'cuentahoracio0', 1800)